conans/paths.py

```diff
@@ -69,10 +69,18 @@ def conan_expand_user(path):
     return os.path.expanduser(path)
 
 
+def get_conan_user_home():
+    tmp = conan_expand_user(os.getenv("CONAN_USER_HOME", "~"))
+    if not os.path.isabs(tmp):
```

Comment:
But, if the user home directory is not an absolute directory... the computer must be very f**, right? How could be relative to anything the user home?